source/styles/home-page.css

```
@@ -61,6 +61,7 @@ img {
   width: 146px;
   position: relative;
   top: 60px;
+  z-index: 10001;
```

Instead of raising the `z-index` to make the Explore button not get blocked by the footer, change this to `margin-bottom: 10vw` instead. The Explore button won't get blocked by the footer in desktop view this way.
```suggestion
  margin-bottom: 10vw;
```